src/windows_registry.rs

```diff
@@ -128,21 +128,28 @@ pub fn find_tool(target: &str, tool: &str) -> Option<Tool> {
         let sub = otry!(lib_subdir(target));
         let (ucrt, ucrt_version) = otry!(get_ucrt_dir());
 
-        let ucrt_include = ucrt.join("Include").join(&ucrt_version);
+        let ucrt_include = ucrt.join("include").join(&ucrt_version);
         tool.include.push(ucrt_include.join("ucrt"));
-        tool.include.push(ucrt_include.join("um"));
-        tool.include.push(ucrt_include.join("winrt"));
-        tool.include.push(ucrt_include.join("shared"));
 
-        let ucrt_lib = ucrt.join("Lib").join(&ucrt_version);
+        let ucrt_lib = ucrt.join("lib").join(&ucrt_version);
         tool.libs.push(ucrt_lib.join("ucrt").join(sub));
 
-        tool.path.push(ucrt.join("bin").join(sub));
-
-        if let Some(dir) = get_sdk10_dir() {
-            tool.libs.push(dir.join("um").join(sub));
-        } else if let Some(dir) = get_sdk81_dir() {
-            tool.libs.push(dir.join("um").join(sub));
+        if let Some((sdk, version)) = get_sdk10_dir() {
```

Comment:
`version` and `ucrt_version` can totally differ. I've seen it happen even on my computer where the max version of ucrt is not the same as the max version of the Windows 10 SDK.